### ISSUE TITLE
Missing parameter error doesn't render HTML

### ISSUE THREAD

**Issue by nijel** [phpmyadmin/phpmyadmin#11872]
The missing parameter error looks weird:

```
schema_export.php: Chybějící parametr: export_type<a href="./doc/html/faq.html#faqmissingparameters" target="documentation"><img src="themes/dot.gif" title="Dokumentace" alt="Dokumentace" class="icon ic_b_help" /></a>```

This is happening in MAINT_4_5_4, QA_4_5 and master, I have not tested other branches.

**Comment by zaverichintan** [comment]
How to reproduce?

**Comment by nijel** [comment]
For example here: https://demo.phpmyadmin.net/master-config/export.php

**Comment by 007durgesh219** [comment]
Hey,  I am able to solve this problem.  Can you please guide me how to submit it.

**Comment by 007durgesh219** [comment]
Hi  @nijel 

To display error  ```PMA_sanitize($error_message)``` function is called in libraries/error.inc.php:56

But according to defniation of PMA_sanitize
```
function PMA_sanitize($message, $escape = false, $safe = false)
{
        if (!$safe) {
            $message = strtr($message, array('<' => '&lt', '>' => '&gt'));
        }

// rest of function.
```

if $safe is not set then it will replace all < with &lt and > with &gt that's why it's not rendering HTML

changing  ```PMA_sanitize($error_message)``` to ```PMA_sanitize($error_message, false, true)``` worked for me.

Am I doing right? Kindly respond

**Comment by zaverichintan** [comment]
The error happens only on some browser and also in some screen resolution.

**Comment by nijel** [comment]
The fix is wrong, as the error message has to be escaped in some cases (eg. when coming from MySQL server or from URL parameters).

**Comment by 007durgesh219** [comment]
can we replace all tags for example all <a> with [a],  <em> with [em].  Like <br /> is replaced by [br/] in error_message (libraries/core.lib.php:228)?.   In that case it won't escape it.

**Comment by 007durgesh219** [comment]
can we replace all tags for example all \<a\> with [a],  \<em\> with [em].  Like \<br /\> is replaced by [br/] in error_message (libraries/core.lib.php:228)?.   In that case it won't escape it.

**Comment by nijel** [comment]
Yes, that's option as well

**Comment by 007durgesh219** [comment]
@nijel So, I have to do that in ```Util::checkParameters``` function before calling ```PMA_fatalError($error_message, null, false);```  (libraries/Util.php:2189). Right?

**Comment by nijel** [comment]
I can see few options:

- make PMA_fatalError accept Message instance, which it would display natively
- change checkParameters to generate bbcode for the error message

**Comment by 007durgesh219** [comment]
won't generating bbcode in checkParameters be a problem in case of ajax request?

**Comment by nijel** [comment]
It would have to be rendered in that case... Probably using Message class is better idea here

**Comment by 007durgesh219** [comment]
@nijel You mean use decodeBB in case of ajax request?

**Comment by 007durgesh219** [comment]
PMA_fatalError is already using Message class in case of ajax request.  So, I think it won't be an issue right?

**Comment by 007durgesh219** [comment]
We need a function which can generate bbcode  similar to as of decodeBB (in Message class). Should I define that also in Message class?

**Comment by 007durgesh219** [comment]
Hi @nijel can you please review if it's fine now?

**Comment by nijel** [comment]
Fixed by merging #11880
